fabfile.py

```diff
@@ -102,7 +106,7 @@ def production():
     """
     Select production instance(s)
     """
-    env.hosts = ['cabot.arachnys.com']
```

Comment:
it was there in master: https://github.com/arachnys/cabot/blame/master/fabfile.py#L101